main.py

```
@@ -1,44 +1,412 @@
+
+# In-memory storage (in production, use a database)
+user_data = {}
+otp_storage = {}
+api_keys = {}
+
+# Helper Functions
+def is_admin(user_id):
+    return user_id in ADMIN_IDS
+
+def generate_otp():
+    return str(random.randint(100000, 999999))
```

Ruff (0.15.7)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)